modules/javafx.controls/src/main/java/javafx/scene/control/skin/TextAreaSkin.java

```diff
@@ -823,14 +837,14 @@ protected int getInsertionPoint(double x, double y) {
         } else {
             Bounds bounds = paragraphNode.getBoundsInLocal();
             double paragraphViewY = paragraphNode.getLayoutY() + bounds.getMinY();
-            if (y >= paragraphViewY
-                    && y < paragraphViewY + paragraphNode.getBoundsInLocal().getHeight()) {
-                return getInsertionPoint(paragraphNode,
-                        x - paragraphNode.getLayoutX(),
-                        y - paragraphNode.getLayoutY());
+            if (
```

Comment:
I really don't like empty opening brackets in conditions but i guess it is not explicitly prohibited. Aside of that fix looks reasonable.